orttraining/orttraining/core/optimizer/compute_optimizer/padding_elimination.cc

```diff
@@ -0,0 +1,550 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <onnx/defs/attr_proto_util.h>
+
+#include "core/framework/random_seed.h"
+#include "core/graph/graph_utils.h"
+#include "core/optimizer/initializer.h"
+#include "orttraining/core/optimizer/compute_optimizer/padding_elimination.h"
+
+using namespace ONNX_NAMESPACE;
```

Comment:
nit: avoid using the whole namespace, append ONNX_NAMESPACE:: before the function that requires it